contrib/endpoints/src/api_manager/api_manager_impl.cc

```diff
@@ -95,7 +101,10 @@ utils::Status ApiManagerImpl::Init() {
       }
     }
 
-    return config_loading_status_;
+    if (global_context_->rollout_strategy().compare(
+            std::string(kConfigRolloutStrategy)) != 0) {
+      return config_loading_status_;
```

Comment:
We don't need check here.  My idea is: you have to make constructing config_manager safe even service_config_ is not empty.   

BTW,  std::string has operator==,  it will be easy to read if using ==